provisioning/app-proxy/kubernetes/templates/cloud/gcp/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: app-proxy
+  namespace: $NAMESPACE
+  labels:
+    app: app-proxy
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+    - host: "hub.${HOSTNAME_SUFFIX}"
+      http:
```

Comment:
There should be two records, `hub....`, `*.hub.`, see: https://kubernetes.io/docs/concepts/services-networking/ingress/#hostname-wildcards